docs/sphinx/data_center_install.rst

```diff
@@ -295,19 +385,13 @@ the `/etc/profile` file:
 
 .. note:: 
 
-  CUDA Quantum is configured to use its own linker, meaning the 
+  The `nvq++` compiler is configured to use its own linker, meaning the 
```

Comment:
`nvq++` is and should be installed such that it is built upon `clang++-16` and, in turn, uses `clang++-16` as its linker driver.

Using `lld` is optional.

Reply:
We've been using lld as linker in our Docker image since maybe March last year. Id is not necessarily installed on the OS, so we do want to include a linker as part of the installation. Is there a reason not to use lld? We could check if the ld command is available and default to, and use lld otherwise, but I also may have been overly cautious with the phrasing here; I am not aware of any cases were the included linker would cause issues.

Reply:
Refreshing my memory on the threads that I read; 
Which linker is used by default for clang depends on the operating system but also on the installed clang package/build. There is already a big conversation around using lld as the default linker [in 2016](https://groups.google.com/g/llvm-dev/c/MualVaiDLfc), and in our build we in fact build dependencies with lld when building with clang. Requests to set lld as default in clang packages are not that uncommon, e.g. [here is one](https://github.com/msys2/MINGW-packages/issues/8253). 

I understand the consideration along the lines of "let the clang ecosystem decide what's best and not force our own opinion on the choice of linker", but unfortunately, I don't think this is practically feasible; the fact that we distribute clang rather than using a package manger installation of clang means that we *have* to make a choice, and that choice potentially deviates from the choice a different clang package makes. Hence, I think making the choice that is consistent across platforms, self-contained (can bootstrap itself), and reproducible is likely a good choice.

I updated the phrasing in the docs.